conda/recipes/kvikio/meta.yaml

```diff
@@ -55,7 +54,7 @@ test:
     - kvikio
 
 about:
-  home: http://rapids.ai/
+  home: https://github.com/rapidsai/kvikio
```

Comment:
We use `https://rapids.ai` for recipes for almost every RAPIDS package that I could find.
```suggestion
  home: https://rapids.ai
```

Reply:
@ajschmidt8 had suggested to add this url, may be because kvikio isn't mentioned on the rapidsai page?

Reply:
whoops, this is my bad. I didn't check the other repositories. I assumed we used the GitHub URLs.

We can switch this back to the main website.